content/2021/theiacon/_index.md

```diff
@@ -0,0 +1,23 @@
+---
+logo: 'images/theiacon-logo.svg'
```

Comment:
this should be images/logos/theiacon-logo.svg

Reply:
Ah, sorry, that's something I forgot to change while trying different things (not the root cause of the issue). 
But thanks a lot for find out the root cause on slack - "2021/theiacon".